test_runner/regress/test_timeline_size.py

```diff
@@ -237,6 +233,9 @@ def test_timeline_physical_size_init(neon_simple_env: NeonEnv):
     env.pageserver.stop()
     env.pageserver.start()
 
+    # FIXME: wait a little, so that the tenant is loaded
```

Comment:
Can we check similar `has_in_progress_downloads` status thing?

Reply:
This isn't actually needed at all. Not yet, anyway; it's needed with PR #2595.